Competition2017/tests/competition/subsystems/collector/MockCollectorSubsystem.java

```diff
@@ -0,0 +1,30 @@
+package competition.subsystems.collector;
+
+import com.google.inject.Inject;
+
+import xbot.common.controls.actuators.XCANTalon;
+import xbot.common.injection.wpi_factories.WPIFactory;
+import xbot.common.properties.DoubleProperty;
+import xbot.common.properties.XPropertyManager;
+
+public class MockCollectorSubsystem extends CollectorSubsystem {
```

Comment:
I think the intent would be clearer here if it were called `TestCollectorSubsystem`: the word "mock" is used elsewhere to refer to versions of classes that don't actually do what they claim to. In this case, it's just a version which is opened up for testing.